### PR TITLE
User can create an entry

### PR DESCRIPTION
**Pull request by nmotamedi**
issue 1

### CODE REVIEW

File: ts/main.ts [hunk 1]
```diff
@@ -1 +1,36 @@
...
+  throw new Error('$urlLinkInput or $entryForm query failed');
+
+$urlLinkInput.addEventListener('input', (event: Event) => {
+  $urlLinkInput.addEventListener('blur', () => {
+    const eventTarget = event.target as HTMLInputElement;
+    if (eventTarget.value.match(/\.(jpeg|jpg|gif|png)$/)) {
+      $previewPhoto?.setAttribute('src', eventTarget.value);
+    }
+  });
+});
```
Comment (shawnkost) [inline ts/main.ts]: You don't want to add another event listener inside of here. This is going to cause you to add a `blur` event listener every single time the user types on their keyboard, so you could be creating hundreds or thousands of events.

I see what you are trying to do here with the `blur` event, but I would just keep the `input` event for the functionality.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One thing here Nader!

**Comment by nmotamedi** [comment]
Screen Shot 2024-02-19 at 2 15 44 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**